alchemist/alchemist-incarnation-biochemistry/src/main/java/it/unibo/alchemist/model/implementations/actions/RemoveJunctionInCell.java

```diff
@@ -35,9 +34,7 @@
 
     private final Junction jun;
     private final Environment<Double> env;
-    @SuppressFBWarnings(value = "SE_BAD_FIELD", justification = "All provided RandomGenerator implementations are actually Serializable")
-    private final RandomGenerator rand;
-    private final ICellNode node;
+    private final CellNode node;
```

Comment:
Not needed. As you did elsewhere, override parent's `getNode()` and cast in there